### ISSUE TITLE
i have not received my funds

### ISSUE THREAD

**Issue by EskimoNorthen** [qtumproject/qtum#918]
I sent my qtum token from binance exchange to my ledger wallet. i cant see the funds in my ledger. Address is correct i can see it here QZ4mF94pDR2Wjv7KqUdWJGMUNgPF7PK3qR but i can not reach my tokens. can anybody help?
Regards

**Comment by JB395** [comment]
Yes, this transaction was successfully sent to the blockchain. Coins are not sent to the wallet, then are always sent as transactions to the blockchain. The wallet manages keys and allows access to send transactions on the blockchain.

Please review the documentation for Ledger Live and using Ledger with QTUM Legacy addresses (which start with a "Q"). This might help https://support.ledger.com/hc/en-us/articles/115003776913-Qtum-QTUM-

Otherwise, please contact Ledger support for help with your Ledger wallet.

**Comment by JB395** [comment]
Alternatively, you can try this approach if you want to restore your Ledger wallet address on Qtum Electrum: https://blog.qtum.org/using-qtum-electrum-with-hardware-wallets-15b91ed693a

**Comment by sergey799** [comment]
Добрый день! Такая же проблема и у меня. Монеты в сети а в Леджер лайф их не видно. Инструкция от специалиста ledger не помогла. 
Ваш кошелек qtum electrum не работает на компьютере. Я его устанавливаю , и пытаюсь запустить. Он пишет вот такое. Что же делать? Не могу управлять своими токенами.
